heartbeat/simple_heart_test.go

```diff
@@ -38,11 +38,12 @@ func (suite *SimpleHeartbeatSuite) TestStrategyIsCalledAtInterval() {
 	strategy.On("Touch", "foo", "bar", suite.Pool).Return(nil)
 
 	h := heartbeat.NewSimpleHeart("bar", "foo", 5*time.Millisecond, suite.Pool, strategy)
+	strategy.AssertNumberOfCalls(suite.T(), "Touch", 1)
```

Comment:
I can't comment on it, but L36 should be changed to `TestStrategyIsCalledAtInitializationAndInterval`. Normally, I'd break this into two tests using `And` as a delimiter, but since `NewSimpleHeart` "begins ticking immediately," this behavior is fine to test together. 😄